Check and balances.md

```diff
@@ -6,16 +6,35 @@ permalink: /checks_and_balances/
 
 # Check and balances : sdg.market
 
-## Selling project parts wihtout owning some
+## Selling project parts without owning some
 In this case. Delijaj is the seller and Nyomi the buyer.
-While testing the trading system on the 3<sup>rd</sup> auction, a seller decided to trad some parts of the project. 
-However, she did not own any part of it but the website let her do it anyway. The parts were at 50 token. A buyer 
+While testing the trading system on the 3<sup>rd</sup> auction, a seller decided to trad some parts of the project.
+However, she did not own any part of it but the website let her do it anyway. The parts were at 50 token. A buyer
```

Comment:
Like said already this is a normal behavior we should remove this from our review

Reply:
oops i actually changed that but as i had issues with the push stuff it let the last version my bad